src/test/ui/deprecation/deprecation-in-future.rs

```diff
@@ -0,0 +1,16 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
```

Comment:
@varkor, should I rename this file to deprecation_in_future-lint.rs or something similar?